test/blackbox-tests/test-cases/cram/run.t

```diff
@@ -25,6 +25,7 @@ Broken phrases don't break the system
 
   $ cat <<EOF
   > Hello, world!
+  > EOF
   Hello, world!
```

Comment:
you can remove this part completely. It was precisely to test the behaviour when the missing `EOF` is present.